test.sh

```diff
@@ -1,36 +1,133 @@
 #!/usr/bin/env bash
+set -ex
 
 function finish {
   echo '-----------------------test.sh------------------------------'
   echo 'Removing test environment'
   echo '------------------------------------------------------------'
   docker-compose down -v
 }
+
 trap finish EXIT
 
 function main() {
-  prepareOutputDir
-  createTestEnvironment
-  loadTestPolicy
-  runTests
-  printProxyConfiguration
-  initializeCert
-  runHTTPSTests
+  runDAP
+  runOSS
+}
+
+# Run DAP Enterprise test suite
+function runDAP() {
```

Comment:
Totally didn't notice this before and you don't have to fix it for this PR but Bash function names use `camel_case` captialization.

Reply:
I changed it (: